packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -222,7 +222,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
           onDismiss={ this.props.onDismiss }
           bounds={ bounds }
           directionalHintFixed={ directionalHintFixed }>
-          <div style={contextMenuStyle} ref={ (host: HTMLDivElement) => this._host = host } id={ id } className={ css('ms-ContextualMenu-container', className) }>
+          <div style={ contextMenuStyle } ref={ (host: HTMLDivElement) => this._host = host } id={ id } className={ css('ms-ContextualMenu-container', className) }>
```

Comment:
just noticed that the structure is `FocusZone` ->` ul` ->`li` . `ul`s as HTML native have `role=list`, we should move `role=menu` and `aria-label` to `ul` instead of `FocusZone`, or mark ul with `role=presentation`